packages/vx-shape/src/shapes/AreaClosed.js

```diff
@@ -13,7 +13,7 @@ export default function AreaClosed({
   strokeDasharray,
   strokeWidth = 2,
   stroke = 'black',
-  fill = rgba(0,0,0,0.3),
+  fill = "rgba(0,0,0,0.3)",
```

Comment:
nice catch. not sure if we've thought about adding `.eslint` for the library but I think [single quotes are preferred](https://github.com/airbnb/javascript#strings--quotes) for strings